test-end-to-end/android-maestro/common/open_pdf_file.yaml

```diff
@@ -0,0 +1,24 @@
+appId: com.toughra.ustadmobile
+
+---
+- tapOn: "Pdf_Content.pdf"
```

Comment:
This does not match the Pdf_Content.pdf file inside test-end-to-end/test-files/content . Where does this text come from? Is it under an open license?

Reply:
Removed this file